dataloader/tasks_segments_parallels.py

```diff
@@ -345,6 +360,57 @@ def load_parallels(json_parallels: [Parallel], db: StandardDatabase) -> None:
             print(f"Could not save parallel {parallel}. Error: ", e)
 
 
+
+def load_parallels_sorted(json_parallels: [Parallel], db: StandardDatabase,filename: str) -> None:
+    """
+    Given an array of parallel objects, load them all into the `sorted parallels` collection presorted.
+
+    :param json_parallels: Array of parallel objects to be loaded as-they-are.
+    :param db: ArangoDB connection object
+
+    """
+
+    db_collection_sorted = db.collection(COLLECTION_PARALLELS_SORTED_BY_FILE)
+    # I wonder if this can be done more efficiently, a lot of spaghetti code...    
+    parallels_sorted_by_src_position = sorted(json_parallels, key=lambda k: k['root_pos_beg'])
+    ids_sorted_by_src_position = []
```

Comment:
we can do this instead of these few lines:

```python
ids_sorted_by_src_position = map(lambda parallel: parallel['id'], parallels_sorted_by_src_position)
```

Reply:
thats a very nice idea indeed, I will do this!